CI/harvester-installer/roles/jenkins/templates/override.conf.j2

```diff
@@ -0,0 +1,7 @@
+# https://www.jenkins.io/doc/book/system-administration/systemd-services/
+[Service]
+Environment="JAVA_OPTS=-Djenkins.install.runSetupWizard=false -Dcasc.jenkins.config=/var/lib/jenkins/casc_configs -Djava.awt.headless=true"
+Environment="CASC_JENKINS_CONFIG=/var/lib/jenkins/casc_configs/"
+{% if JENKINS_USING_PROXY %}
+Environment="JENKINS_OPTS=--httpListenAddress=127.0.0.1"
+{% endif %}
```

Comment:
Needs a newline at end of file

Reply:
Good catch, I've updated that, thanks @noahgildersleeve :smile: